acceptance/config/ec2-west-el6-64mda-ubuntu1204-64a.cfg

```diff
@@ -0,0 +1,28 @@
+HOSTS:
+  el6-64-1:
```

Comment:
as discussed, the old el5 stuff was used by the 3 node dup_collected_resources.rb, we'll need a new 3 node combination. I'd suggest having 2 el6 nodes or something. Maybe an el6 and el7 or some such, not too fussed.